pom.xml

```diff
@@ -51,7 +51,7 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <zookeeper.version>3.5.1-alpha</zookeeper.version>
-    <bookkeeper.version>4.3.4-TWTTR</bookkeeper.version>
+    <bookkeeper.version>4.3.4-TWTTR-SNAPSHOT</bookkeeper.version>
```

Comment:
This is to make sure that it takes this particular change: https://github.com/twitter/bookkeeper/pull/2 into account. But this works without this change too. Will update the pull request with all the comments taken care of.